README.md

```diff
@@ -1,85 +1,236 @@
 # Terraform Bridge Provider Boilerplate
 
-This repository contains boilerplate code for building a new Pulumi provider which wraps an existing
-Terraform provider, if the existing provider uses _Go Modules_.
+This repository contains boilerplate code for building a new Pulumi provider which wraps an existing Terraform provider.  These instructions are primarily intended for internal use by Pulumi as we have not yet refined the process for general consumption by the community at large, but this document may serve as a rough guide for community members who want to create their own Pulumi providers that wrap an existing Terraform provider.
 
-Modify this README to describe:
+## Creating a Pulumi Terraform Bridge Provider
 
-- The type of resources the provider manages
-- Add a build status image from Travis at the top of the README
-- Update package names in the information below
-- Add any important documentation of concepts (e.g. the "serverless" components in the AWS provider).
+The following instructions assume a Pulumi-owned provider based on an upstream provider named `terraform-provider-foo`.  Substitute appropriate values below for your use case.
 
-## Creating a Pulumi Terraform Bridge Provider
+> Note: If the name of the desired Pulumi provider differs from the name of the Terraform provider, you will need to carefully distinguish between the references - see <https://github.com/pulumi/pulumi-azure> for an example.
+
+### Prerequisites
+
+Ensure [`pulumictl`](https://github.com/pulumi/pulumictl#installation) is installed.
+
+### Creating and Initializing the Repository
+
+Pulumi offers this repository as a [GitHub template repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template) for convenience.  From this repository:
+
+1. Click "Use this template".
+1. Set the following options:
+    - Owner: pulumi (or your GitHub organization/username)
+    - Repository name: pulumi-foo
+    - Description: Pulumi provider for Foo
+    - Repository type: Public
+1. Clone the generated repository to the appropriate location in your `$GOPATH`.
+
+From the templated repository:
+
+1. Create a branch on which to do this work:
+
+    ```bash
+    git checkout -b initial-scaffold
+    ```
```

Comment:
What's the goal of this?  If it is needed, should there be a step that merges this branch back?

Reply:
I'm assuming that most repos do not allow pushing directly on the main branch.  There's a step at the end that instructs the user to create a PR, but happy to delete if you think it's unnecessary since the doc is relatively long to begin with.

What do you think?

Reply:
@stack72 Could you chime in here as well?  I'm not attached to that step being present if we feel it's either incorrect, unnecessary, or it's fairly obvious that this should be done on a branch.

Reply:
Removed.  Marking resolved.